watertap/property_models/seawater_prop_pack.py

```diff
@@ -704,6 +705,7 @@ def define_metadata(cls, obj):
                 "therm_cond_phase": {"method": "_therm_cond_phase"},
                 "dh_vap_mass": {"method": "_dh_vap_mass"},
                 "diffus_phase_comp": {"method": "_diffus_phase_comp"},
+                "bpe_phase": {"method": "_bpe_phase"},
```

Comment:
I would suggest a more descriptive name for this: `boiling_point_elevation`. It is longer and more cumbersome, but it is much more informative to someone trying to use the property package.

Reply:
I agree. I also wonder if this needs `_phase` in the name. I think not, unless there is an IDAES standard that _does_ include `phase` as a qualifier.

Reviewing the IDAES standards (https://idaes-pse.readthedocs.io/en/1.4.0/standards.html), it seems that we have no restriction to abide by, so perhaps just `boiling_point_elevation` would be fine without the phase qualifier and corresponding index.

Reply:
I did ask this question myself - for this application it is fairly obvious that there is only one BPE value. However, there are cases where you might have two liquid phases with different BPE values - these are probably not on the immediate horizon for WaterTap, but it might be worth thinking ahead a little in this case.

Reply:
Ah, I didn't think of that!